servers/src/main/java/tachyon/master/file/options/CreateDirectoryOptions.java

```diff
@@ -17,15 +17,14 @@
 
 import tachyon.conf.TachyonConf;
 import tachyon.master.MasterContext;
-import tachyon.thrift.MkdirTOptions;
+import tachyon.thrift.CreateDirectoryTOptions;
 
 /**
  * Method option for creating a directory.
  */
-public final class MkdirOptions {
-
+public final class CreateDirectoryOptions {
   /**
-   * Builder for {@link MkdirOptions}.
```

Comment:
I would prefer to use `CreateFile` and `CreateDirectory` to keep the 1-to-1 correspondence, do you mind if we refactor this afterward?